docker-compose.yml

```diff
@@ -4,12 +4,13 @@ services:
     image: sethmachineio/valheim-server
     user: "1000:1000"
     ports:
-      - 2456:2456
-      - 2457:2457
-      - 2458:2458
+      - 2456:2456/udp
+      - 2457:2457/udp
+      - 2458:2458/udp
```

Comment:
Great for catching this, I haven't worked with the `docker-compose.yml` files before and didn't know if you have to specify UDP otherwise it will just default to TCP (in which case it wouldn't work, right?).

Reply:
It would expose both, tcp and udp.
but why should be expose more than we need, right?